db/seeders/20220303224613-dummy-patient.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+/* eslint-disable no-unused-vars */
+const { uuid } = require('uuidv4')
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
For a follow up PR we should make the dummy data a bit more diverse.

Reply:
Added issue: https://github.com/healthy-queue/back-end/issues/18 to account for this work.